### ISSUE TITLE
Help!

### ISSUE THREAD

**Issue by darshansj01** [pierre42100/WEB-centralAuthServer#2]
Hey there.

Thanks for uploading this project.

I really need help to use it.

**Comment by pierre42100** [comment]
Did you manage to install it ?

**Comment by darshansj01** [comment]
NO. I need a complete guide on how to proceed with the project.

darshansj01@gmail.com

Could you please help me out.
